### ISSUE TITLE
Fix problems found by accessibility report

### ISSUE THREAD

**Issue by Macroz** [CSCfi/rems#2431]
The current automated accessibility report contains major / critical violations. It is useful to try to fix them directly or create new specific tickets out of the difficult ones.

- [ ] fix found errors so that there is empty violations.json / tickets created for each leftover
